libs/sdk-ui-charts/src/highcharts/chart/highcharts/test/customConfiguration.test.ts

```diff
@@ -183,6 +184,31 @@ describe("getCustomizedConfiguration", () => {
             expect(result.xAxis[0]).toEqual(expectedResult);
         });
 
+        it.each([
+            ["set", "enabled", { chart: { zoomType: "x" } }],
+            ["not set", "disabled", {}],
+        ])(
+            "should %s X axis configurations with the minRange property when the zooming is %s",
+            (_desc1: string, _desc2: string, chartConfig: Partial<IChartConfig>) => {
+                const { chart } = chartConfig;
+                const result = getCustomizedConfiguration(
+                    {
+                        ...chartOptions,
+                    },
+                    {
+                        chart,
+                    },
+                );
+
+                const expectedResult = {
+                    ...result.xAxis[0],
+                    minRange: chart?.zoomType ? 2 : undefined,
```

Comment:
Adding logic inside a parametrized tests hurts maintainability (what if we were to add more scenarios? then we would need to change this condition unexpectedly). Can we add the expected result in the scenarios? Or use snapshots?

Reply:
I make a separate to another case, hope this is more clear and easy to maintain. Please help review this change.